src/components/pageheader/PageHeader.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './pageheader.less';
+import { Typography } from 'antd';
+const { Title, Paragraph } = Typography;
+
+type PageHeaderProps = {
```

Comment:
Prefer to use interfaces for props 
```suggestion
interface PageHeaderProps {
```